gatsby/plugins/gatsby-source-bnf/src/field-extensions/anchors/anchor-replacer.test.ts

```diff
@@ -18,98 +18,257 @@ describe("replaceInternalAnchors", () => {
+					nodeModel
+				)
+			).toBe(
+				`<a href="/nurse-prescribers-formulary/#medicinal-preparations" title="NPF Medicinal Preparations">NPF Medicinal Preparations</a>`
+			);
 		});
-		expect(
-			replaceInternalAnchors(
-				[
-					`<a href="https://www.nice.org.uk">external</a>`,
-					`<a href="/drugs/a-drug/" title="A drug">drug</a>`,
-					`<a href="https://www.nice.org.uk">external</a>`,
-				].join(""),
-				nodeModel
-			)
-		).toBe(
-			`<a href="https://www.nice.org.uk">external</a><a href="/drugs/a-drug/" title="A drug">drug</a><a href="https://www.nice.org.uk">external</a>`
-		);
 	});
 
-	it("should throw if node can't be found by id", () => {
-		getNodeById.mockReturnValueOnce(null);
-		expect(() => {
-			replaceInternalAnchors(
-				`a <a href="/#/content/bnf/_123" title="A drug">drug</a> link`,
-				nodeModel
+	describe("Wound management", () => {
+		it("should replace link to wound management index without section hash", () => {
+			getNodeById.mockReturnValueOnce({
+				title: "Wound management products and elasticated garments",
+				internal: { type: BnfNode.WoundManagementIntroduction },
+			});
+
+			expect(
+				replaceRelativeAnchors(
+					`<a title="Wound management products and elasticated garments" href="/woundManagement/_320704649">Wound management products and elasticated garments</a>`,
+					nodeModel
+				)
+			).toBe(
+				`<a title="Wound management products and elasticated garments" href="/wound-management/">Wound management products and elasticated garments</a>`
 			);
-		}).toThrow(
-			`Couldn't find node with id _123 in anchor <a href="/#/content/bnf/_123" title="A drug">drug</a>`
-		);
+		});
+
+		it("should replace link to wound management index with section hash", () => {
+			getNodeById.mockReturnValueOnce({
+				title: "Wound management products and elasticated garments",
+				sections: [
+					{
+						title: "Introduction",
+						id: "section_320704649-0",
+					},
+				],
+				internal: { type: BnfNode.WoundManagementIntroduction },
+			});
+			expect(
+				replaceRelativeAnchors(
+					`<a title="Wound management products and elasticated garments" href="/woundManagement/_320704649#section_320704649-0">Wound management products and elasticated garments</a>`,
+					nodeModel
+				)
+			).toBe(
+				`<a title="Wound management products and elasticated garments" href="/wound-management/#introduction">Wound management products and elasticated garments</a>`
+			);
+		});
+
+		it("should replace link to wound management taxonomy", () => {
+			getNodeById.mockReturnValueOnce({
+				title: "Low adherence dressing",
+				internal: { type: BnfNode.WoundManagementTaxonomy },
+			});
+
+			expect(
+				replaceRelativeAnchors(
+					`<a href="/woundManagement/taxonomy/_870221431">Low adherence dressing</a>`,
+					nodeModel
+				)
+				// TODO Update this path here when we know what URL structure wound management will have
```

Comment:
Just double checking that we don't know this as yet?

Reply:
Yup it's in https://nicedigital.atlassian.net/browse/BNF-1239